explorer/src/Pos/Explorer/Socket/Methods.hs

```diff
@@ -358,10 +359,10 @@ getBlundsFromTo
     :: forall ctx m . ExplorerMode ctx m
     => HeaderHash -> HeaderHash -> m (Maybe [Blund])
 getBlundsFromTo recentBlock oldBlock =
-    DB.getHeadersRange Nothing oldBlock recentBlock >>= \case
+    tryAny (DB.getHashesRange Nothing oldBlock recentBlock) >>= \case
```

Comment:
Perhaps it would be better to not catch the exception? But right, `getHashesRange` is given a type that doesn't even admit the possibility of a normal failure. I'll write about that in a separate comment.

I take issue with this one because we're conflating two very different reasons for failure:

1. Something normal/expected, like not knowing one of the endpoints.
2. Something *actually* exceptional, like the database being corrupt or a failure to open some file it uses.